fe/fe-core/src/main/java/com/starrocks/sql/optimizer/OptimizerTraceUtil.java

```diff
@@ -75,6 +75,16 @@ public static void logMVRewrite(String mvName, String format, Object... objects)
         });
     }
 
+    /**
+     * NOTE: Carefully use it, because the log would be print into the query profile, to help understanding why a
+     * materialized view is not chose to rewrite the query.
+     */
+    public static void logMVRewriteFailReason(String mvName, String format, Object... objects) {
```

Comment:
what is the difference between trace logs mv with trace reason mv?

Reply:
- trace logs: a lot of verbose log about mv rewriting, including internal states
- trace reason: only explain why cannot rewrite this mv, it is supposed to be understandable by end user